laravel/app/Http/Controllers/PagesController.php

```diff
@@ -166,23 +166,26 @@ private function processChoices($date, $id)
 
         if ($schedule1 != null) {
             $first_choice = self::processSingleChoice($schedule1);
-            $day_arr['first'] = "First Choice: $first_choice\nMilestone: $milestone1C - $milestone1D\nObjective: $objective1\nAnesthesiologist Preference: $pref_anest1";
+            $first_choice = "First Choice: $first_choice\nMilestone: $milestone1C - $milestone1D\nObjective: $objective1\nAnesthesiologist Preference: $pref_anest1";
```

Comment:
What is the reason for splitting this into 2 lines?

Reply:
prettier splits `$day_arr['first']` into
```php
$day_arr[
    'first'
]
```
if I do keep it as one line. the alternative is a `// prettier-ignore` comment

Reply:
Since it would need a `// prettier-ignore` comment, I don't really have preference.

Reply:
great! unless @Spensenaz has a preference, I'll leave it as is.